### DISCUSSION TITLE
Multihoming section?

### DISCUSSION THREAD

**Discussion by becarpenter** [becarpenter/book6#79] [Ideas]
Should we have a separate section in Management and Operations on multihoming? (There is already a section on multi-address operation, but that is very specific and makes it clear that MHMP is an open issue.) And what can we say that is _useful_ and _non-controversial_?

**Comment by joaolucasmacedo** [comment]
Sure. There is a recent RFC [8678](https://www.rfc-editor.org/rfc/rfc8678.html) on it.
I wrote a article about that issue, maybe you guys can comment and add to the book, if you find useful. 
https://www.linkedin.com/pulse/ipv6-nat-eating-crow-jo%25C3%25A3o-lucas-macedo/

**Comment by becarpenter** [comment]
There is also a complex discussion in the IETF about https://datatracker.ietf.org/doc/draft-fbnvv-v6ops-site-multihoming/, so until that settles down, it is hard to write a non-controversial summary. Thanks for your pointer, will read.

**Reply by buraglio** [reply]
Yes, as a co-author of that I can say that the largest sticking points are (generally), the NAT66 and NPTv6 sections. Having tested both those scenarios and actively using both methods on production, I can say that they "work", but they definitely violate some saved tenants. Perhaps we break out the types of multihoming first (load sharing and active/backup) and wait to see what shakes out. If I get some time I will try to get some text in there
